plugins/jira-dashboard/src/components/JiraTable/columns.tsx

```diff
@@ -75,18 +75,30 @@ export const columns: TableColumn[] = [
   },
   {
     title: 'Assignee',
-    field: 'fields.assignee.name',
+    field: 'fields.assignee.displayName',
     highlight: true,
     type: 'string',
     width: '10%',
 
     render: (issue: Partial<Issue>) => {
-      if (issue.fields?.assignee?.name) {
+      if (issue.fields?.assignee?.displayName) {
+        return (
+          <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
Can we use `Avatar` from `backstage/core-components` for consistency? I prefer the rounder look that is used elsewhere :)

Reply:
Sure

Reply:
Updated it